### PR TITLE
implement [Alerts Info] add notification tab

### PR DESCRIPTION
**Pull request by adi-gini**
*Jira link* :
https://iguazio.atlassian.net/browse/ML-8618

Screenshot 2024-12-19 at 15 11 19

### CODE REVIEW

File: src/components/DetailsInfo/detailsInfo.scss [hunk 1]
```diff
@@ -168,5 +169,5 @@ $itemInfoWithoutPadding: item-info-without-padding;
 }
 
 :export {
-  itemInfoWithoutPadding: $itemInfoWithoutPadding;
+  iteminfowithoutpadding: $itemInfoWithoutPadding;
```
Comment (mariana-furyk) [inline src/components/DetailsInfo/detailsInfo.scss]: why did you rename variable?

File: src/components/DetailsInfo/detailsInfo.util.js [hunk 2]
```diff
@@ -206,13 +205,19 @@ export const generateAlertsDetailsInfo = pageData => {
...
-        key={valueItem.tooltip + index}
-        template={<TextTooltipTemplate text={valueItem.tooltip} />}
-      >
-        {valueItem.icon}
-      </Tooltip>
+    AlertsDetailsInfo.notificationsDetailsInfo = notifications.map((notification, index) => (
+      <li className="notifications-item" key={index}>
+        <div className="notifications-item_icon">{notification.icon}</div>
+        <div>
+          <div className="notifications-item_header">{upperFirst(notification.kind)}</div>
```
Comment (mariana-furyk) [inline src/components/DetailsInfo/detailsInfo.util.js]: we add only modifiers with "_", I believe "header" is not a modifier

File: src/components/DetailsInfo/detailsInfo.util.js [hunk 3]
```diff
@@ -206,13 +205,19 @@ export const generateAlertsDetailsInfo = pageData => {
...
+          <div className="notifications-item_header-text">
+            {`${notification.succeeded} success `}
+            <span className={notification.failed > 0 ? 'text-red' : ''}>
```
Comment (mariana-furyk) [inline src/components/DetailsInfo/detailsInfo.util.js]: use classnames library, also rename text-red to notifications-item__header-text_failed

File: src/components/DetailsInfo/detailsInfo.util.js (hunk 2 above)
Comment (adi-gini) [reply src/components/DetailsInfo/detailsInfo.util.js]: Done

File: src/components/DetailsInfo/detailsInfo.util.js (hunk 3 above)
Comment (adi-gini) [reply src/components/DetailsInfo/detailsInfo.util.js]: Done

File: src/components/DetailsInfo/detailsInfo.scss (hunk 1 above)
Comment (adi-gini) [reply src/components/DetailsInfo/detailsInfo.scss]: It happens automatically when I use Prettier

Comment (mariana-furyk) [reply src/components/DetailsInfo/detailsInfo.scss]: change it back please

File: src/components/DetailsInfo/detailsInfo.util.js [hunk 4]
```diff
@@ -206,13 +206,23 @@ export const generateAlertsDetailsInfo = pageData => {
...
-      </Tooltip>
+    AlertsDetailsInfo.notificationsDetailsInfo = notifications.map((notification, index) => (
+      <li className="notifications-item" key={index}>
+        <div className="notifications-item_icon">{notification.icon}</div>
+        <div>
+          <div className="notifications-item-header">{upperFirst(notification.kind)}</div>
+          <div className="notifications-item-header-text">
+            {`${notification.succeeded} success `}
+            <span
+              className={classNames('notifications-item-header-text_failed', {
```
Comment (mariana-furyk) [inline src/components/DetailsInfo/detailsInfo.util.js]: no need to use {}, you can add notification.failed > 0 && notifications-item-header-text_success

File: src/components/DetailsInfo/detailsInfo.util.js [hunk 5]
```diff
@@ -206,13 +206,23 @@ export const generateAlertsDetailsInfo = pageData => {
...
-        key={valueItem.tooltip + index}
-        template={<TextTooltipTemplate text={valueItem.tooltip} />}
-      >
-        {valueItem.icon}
-      </Tooltip>
+    AlertsDetailsInfo.notificationsDetailsInfo = notifications.map((notification, index) => (
+      <li className="notifications-item" key={index}>
+        <div className="notifications-item_icon">{notification.icon}</div>
+        <div>
+          <div className="notifications-item-header">{upperFirst(notification.kind)}</div>
```
Comment (mariana-furyk) [inline src/components/DetailsInfo/detailsInfo.util.js]:
```suggestion
          <div className="notifications-item__header">{upperFirst(notification.kind)}</div>
```

**Review by mariana-furyk — APPROVED** [review]
